exercises/sum-of-multiples/sum_of_multiples.example

```diff
@@ -0,0 +1,14 @@
+factored([Curr|Factors], Num) :-
+    0 is mod(Num, Curr), !;
+    factored(Factors, Num).
+
+sum_of_multiples(_, 0, 0) :- !.
+sum_of_multiples(Factors, Limit, Sum) :-
+    Next is Limit - 1,
+    sum_of_multiples(Factors, Next, NextSum),
```

Comment:
The style guide I referenced suggests the `NextSum` should be `Next_Sum`. However, I like CamelCase better than Whatever_Case_This_Is. What do you think?